src/main/scala/org/broadinstitute/dsde/firecloud/service/FireCloudDirectives.scala

```diff
@@ -36,19 +36,18 @@ trait FireCloudDirectives extends spray.routing.Directives with PerRequestCreato
   def encodeUri(path: String): String = FireCloudDirectiveUtils.encodeUri(path)
 
   private def generateExternalHttpPerRequestForMethod(uri: Uri, inMethod: HttpMethod) = {
-    val outMethod = new RequestBuilder(inMethod)
     // POST, PUT, PATCH
     if (inMethod.isEntityAccepted) {
```

Comment:
oddly,` HttpMethods.GET.isEntityAccepted` returns `true` now (and DELETE as well)

```
scala> HttpMethods.GET.isEntityAccepted
res3: Boolean = true
```

looking into the akka-http implementation, GET and DELETE "tolerate" entities, and for the other methods it's "expected". if we want to be tolerant of these potential cases, i propose removing the branching here

we could alternatively keep the branching and call `getRequestEntityAcceptance` on `inMethod` and see if the value is `Expected`